### PR TITLE
Add GCE handling to uploader service.

### PR DESCRIPTION
**Pull request by smarlowucf**

### CODE REVIEW

File: mash/services/uploader/cloud/gce.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+        self.config = get_configuration(service='uploader')
+
+    def upload(self):
+        storage_driver = GoogleStorageDriver(
+            self.credentials['client_email'],
+            secret=self.auth_file.name,
+            project=self.credentials['project_id']
+        )
+
+        object_name = os.path.basename(self.system_image_file)
```
Comment (rjschwei) [inline mash/services/uploader/cloud/gce.py]: At least for releases we want the uploaded tarball to have the same name as the image we release.

**Comment by smarlowucf** [comment]
@rjschwei Fixed tarball name. Also accounted for family and date format in image descriptions.

File: mash/services/uploader/cloud/gce.py [hunk 2]
```diff
@@ -39,7 +39,8 @@ class UploadGCE(UploadBase):
     .. code:: python
 
         custom_args={
-            'bucket': 'images'
+            'bucket': 'images',
+            'family': 'sles-11',
```
Comment (rjschwei) [inline mash/services/uploader/cloud/gce.py]: Another sles-11 reference ;)

Comment (smarlowucf) [reply mash/services/uploader/cloud/gce.py]: Cleaned up.

**Pull request merged**